client/dive-common/components/ImportAnnotations.vue

```diff
@@ -133,16 +149,33 @@ export default defineComponent({
             target="_blank"
           >Data Format Documentation</a>
         </v-card-text>
-        <v-card-actions>
-          <v-btn
-            depressed
-            block
-            :disabled="!datasetId || processing"
-            @click="openUpload"
-          >
-            Import
-          </v-btn>
-        </v-card-actions>
+        <v-container>
+          <v-col>
+            <v-row>
+              <v-btn
+                depressed
+                block
+                :disabled="!datasetId || processing"
+                @click="openUpload"
+              >
+                Import
+              </v-btn>
+            </v-row>
+            <v-row>
+              <v-switch
+                v-model="additive"
+                label="Additive"
+              />
```

Comment:
I feel like `additive` isn't intuitive. Perhaps instead a checkbox with an 'overwrite' label defaulted to checked.